app/konnect/analytics/custom-reports/use-cases.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Use Cases
```

Comment:
```suggestion
title: Report Use Cases
```
I think "Use Cases" would probably be fine in the nav bar, but I think expanding the title here would help for searches on Google/help users who come here from a search.